iis.json

```diff
@@ -0,0 +1,78 @@
+{
+   "1": {
```

Comment:
What are the numbers for? Is it only for preserving the order? If so, you better use json list collection.